caas/kubernetes/provider/k8snamespaces.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2019 Canonical Ltd.
```

Comment:
just need to call this file namespaces.go
it's already in a kubernetes package

Reply:
renamed the file